lib/msfdb_helpers/pg_ctlcluster.rb

```diff
@@ -0,0 +1,140 @@
+require 'msfdb_helpers/db_interface'
+
+class PgCtlcluster < DbInterface
```

Comment:
Just confirming, should these be in a MsfdbHelpers module/namespace or similar? 👀

Reply:
probably should be, will do